docs/source/guide/explanation/algorithms/segmentation/instance_segmentation.rst

```diff
@@ -92,11 +92,52 @@ Recently, we have made updates to MaskRCNN-ConvNeXt, incorporating the `ConvNeXt
 
 .. To be added soon
 
-.. ************************
-.. Semi-supervised Learning
-.. ************************
+************************
+Semi-supervised Learning
+************************
+
+We employ the modified `Unbiased Teacher framework <https://arxiv.org/abs/2102.09480>`_ to tackle the problem of :ref:`Semi-supervised learning <semi_sl_explanation>` in instance segmentation.
+This framework leverages two models during training: a "student" model, which serves as the primary model being trained, and a "teacher" model, which acts as a guiding reference for the student model.
+
+During training, the student model is updated using both ground truth annotations (for labeled data) and pseudo-labels (for unlabeled data).
+These pseudo-labels are generated by the teacher model's predictions. Notably, the teacher model's parameters are updated based on the moving average of the student model's parameters.
+This means that backward loss propagation is not utilized for updating the teacher model. Once training is complete, only the student model is used for making predictions in the instance segmentation task.
+
+We also use the warmup stage for the teacher model during the first epochs to avoid utilizing too misleading pseudo labeling. We use constant thresholding for pseudo bounding boxes and, unlike in Unbiased Teacher work, we utilize all unlabeled losses including regression one and mask loss.
+There are some key differences in the augmentation pipelines used for labeled and unlabeled data.
+Basic augmentations such as random flip, random rotate, and random crop are employed for the teacher model's input.
+On the other hand, stronger augmentations like color distortion, RGB to gray conversion, Gaussian blur and `Random Erasing <https://arxiv.org/abs/1708.04896>`_ are applied to the student model.
+This discrepancy helps improve generalization and prevents unnecessary overfitting on the pseudo-labels generated by the teacher model.
+As well as for the supervised pipeline we utilize EMA smoothing for the student model during warm-up and the rest of the training.
+
+.. note::
+
+    To fine-tune better on small labeled datasets used in Semi-SL tasks we adopted a repeat dataset which brought metric improvement in our experiments. However, the training time also increased noticeably.
```

Comment:
To obtain a better performance after fine-tuning, ...
... we adopt ..., which brings ... increases -- for consistency with other sentences